test-utils/index.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+const ethers = require('ethers');
+const {privateToAddress} = require('ethereumjs-util');
+const {hash, prefix0x, fromRpcSig} = require('../lib/utils');
+const proxyquire = require('proxyquire').noPreserveCache().noCallThru();
+
+const Wallet = proxyquire('../lib/wallet/wallet', {
+    './client-fund-contract'    : function() {
+        return {};
+    },
+    './balance-tracker-contract': function() {
+        return {};
+    },
+    './erc20-contract'          : function() {
+        return {};
+    }
+});
+
+class ApiPayloadFactory {
+    constructor(options) {
+        this.amount = options.amount;
+        this.currency = options.currency;
+
+        this.senderPrivateKey = options.senderPrivateKey;
+        this.sender = options.sender || this.senderPrivateKey ?
+            prefix0x(privateToAddress(Buffer.from(this.senderPrivateKey, 'hex')).toString('hex')) :
+            undefined;
+        this.recipient = options.recipient;
+        this.operatorPrivateKey = options.operatorPrivateKey;
+        this.operator = options.operator || this.operatorPrivateKey ?
+            prefix0x(privateToAddress(Buffer.from(this.operatorPrivateKey, 'hex')).toString('hex')) :
+            undefined;
+    }
+
+    createUnsignedPayment(senderRef) {
+        const senderData = Buffer
+            .from(JSON.stringify({ref: senderRef}))
+            .toString('base64');
+
+        return {
```

Comment:
Not a huge fan of this kind of formatting. Looks messy to me.

Reply:
You're thinking of the : alignment? I noticed this now as well, I think this is because I used a different machine/setup than I usually use for editing. Will clean it up on the next pass.